scripts/mongo/createSearchDocuments.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+$options = getopt(
+    "c:s:q:hd:i:a",
+    array(
+        "config:",
+        "storename:",
+        "tripod-dir:",
+        "arc-dir:",
+        "spec:",
+        "id:",
+        "help",
+        "stat-loader:",
+        "queue",
+        "async"
+    )
+);
+
+function showUsage($scriptName)
+{
+    $help = <<<END
+Usage:
+
+php $scriptName -c/--config path/to/tripod-config.json -s/--storename store-name [options]
+
+Options:
+    -h --help               This help
+    -c --config             path to MongoTripodConfig configuration (required)
+    -s --storename          Store to create views for (required)
+    -d --spec               Only create for specified search document specs
+    -i --id                 Resource ID to regenerate search documents for
+    -a --async              Generate table rows via queue
+    -q --queue              Queue name to place jobs on (defaults to configured TRIPOD_APPLY_QUEUE value)
```

Comment:
You don't - the purpose is you'd fire up a worker to listen for that queue to keep it from polluting/clogging the main queue